### PR TITLE
Add acceptance test to view account information on the account page

### PR DESCRIPTION
**Pull request by jasson99**
## Description
Add acceptance test to view account information on the account page on the webUI 

## Related Issue

- Related to issue #2566 

## Motivation and Context
The feature to view the account information on the account page was implemented in the phoenix but no tests were written for that feature. This PR adds acceptance test scenarios for that feature implemented in phoenix.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/acceptance/features/webUIAccount/accountInformation.feature [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Feature: View account information
+  As a user
+  I would like to view account information
```
Comment (HariBhandari07) [inline tests/acceptance/features/webUIAccount/accountInformation.feature]: ```I would like to view my account information```

File: tests/acceptance/features/webUIAccount/accountInformation.feature [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  Scenario: view account information when the user has been created without groups membership
+    Given user "user1" has logged in using the webUI
+    When the user browses to the account page
+    Then the user should have following details displayed on the account information
+      | Username          | user1                         |
+      | Display name      | User One                      |
+      | Email             | user1@example.org             |
+      | Groups membership | You are not part of any group |
+
+  Scenario: view account information when the user has been created with a group membership
```
Comment (HariBhandari07) [inline tests/acceptance/features/webUIAccount/accountInformation.feature]: ```view account information when the user has been added to a group```

File: tests/acceptance/pageObjects/accountPage.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+    /**
+     * Gets the account information details from the account information element on the account page
+     *
+     * @returns {object}
+     */
+    getAccountInformation: async function () {
+      const accountInformation = []
+      let accountInfoElementIds = []
+      await this.waitForElementVisible('@accountInformationElement')
```
Comment (HariBhandari07) [inline tests/acceptance/pageObjects/accountPage.js]: await is not required here

File: tests/acceptance/pageObjects/accountPage.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      await this.api.elements('@accountInformationElement', function (result) {
+        accountInfoElementIds = result.value
+      })
+      for (const i in accountInfoElementIds) {
```
Comment (HariBhandari07) [inline tests/acceptance/pageObjects/accountPage.js]: please use meaningful variable name in place of ```i```

File: tests/acceptance/pageObjects/accountPage.js (hunk 3 above)
Comment (jasson99) [reply tests/acceptance/pageObjects/accountPage.js]: why?

File: tests/acceptance/pageObjects/accountPage.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
+module.exports = {
+  url: function () {
+    return this.api.launchUrl + '/index.html#/account/'
+  },
+  commands: {
+    /**
+     * like build-in navigate() but also waits till for the progressbar to appear and disappear
+     * @returns {*}
```
Comment (dpakach) [inline tests/acceptance/pageObjects/accountPage.js]:
```suggestion
     * @returns {Promise}
```

File: tests/acceptance/pageObjects/accountPage.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+      }
+      return actualAccInfo
+    }
+  },
+  elements: {
+    accountDisplay: {
+      selector: '//div//span[@data-msgid="Account"]',
+      locateStrategy: 'xpath'
+    },
+    accountInformationElement: {
```
Comment (dpakach) [inline tests/acceptance/pageObjects/accountPage.js]:
```suggestion
    accountInformationElements: {
```

File: tests/acceptance/pageObjects/accountPage.js [hunk 7]
```diff
@@ -0,0 +1,51 @@
+module.exports = {
+  url: function () {
+    return this.api.launchUrl + '/index.html#/account/'
+  },
+  commands: {
+    /**
+     * like build-in navigate() but also waits till for the progressbar to appear and disappear
+     * @returns {*}
+     */
+    navigateTo: function () {
```
Comment (dpakach) [inline tests/acceptance/pageObjects/accountPage.js]:
```suggestion
    navigateAndWaitTillLoaded: function () {
```

File: tests/acceptance/features/webUIAccount/accountInformation.feature [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+  Scenario: view account information when the user has been created without groups membership
+    Given user "user1" has logged in using the webUI
+    When the user browses to the account page
+    Then the user should have following details displayed on the account information
+      | Username          | user1                         |
+      | Display name      | User One                      |
+      | Email             | user1@example.org             |
+      | Groups membership | You are not part of any group |
+
+  Scenario: view account information when the user has been added to a group
```
Comment (individual-it) [inline tests/acceptance/features/webUIAccount/accountInformation.feature]: try also a group membership of multiple groups

Comment (jasson99) [reply tests/acceptance/features/webUIAccount/accountInformation.feature]: done

File: tests/acceptance/stepDefinitions/accountContext.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+})
+
+Then('the user should have following details displayed on the account information', async function (dataTable) {
+  const expectedAccInfo = dataTable.rowsHash()
+  const actualAccInfo = await client.page.accountPage().getAccountInformation()
+  const actualEntries = Object.entries(actualAccInfo)
+  for (const [key, value] of actualEntries) {
+    if (key === 'Groups membership') {
+      const differenceInGroups = _.difference(value.split(',').map(grp => grp.trim()),
+        expectedAccInfo[key].split(',').map(grp => grp.trim()))
```
Comment (skshetry) [inline tests/acceptance/stepDefinitions/accountContext.js]: Please fix the formatting in a way it's more easy to follow.

File: tests/acceptance/features/webUIAccount/accountInformation.feature [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+    And user "user1" has been added to group "Group31"
+    And user "user1" has been added to group "A111111"
+    And user "user1" has logged in using the webUI
+    When the user browses to the account page
+    Then the user should have following details displayed on the account information
+      | Username          | user1                                                            |
+      | Display name      | User One                                                         |
+      | Email             | user1@example.org                                                |
+      | Groups membership | Group1, Group2, Group3, Group4, Group5, Group6, Group31, A111111 |
+
```
Comment (skshetry) [inline tests/acceptance/features/webUIAccount/accountInformation.feature]: Remove extra line
```suggestion
```

File: tests/acceptance/features/webUIAccount/accountInformation.feature [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+      | Group31   |
+      | A111111   |
+    And user "user1" has been added to group "Group1"
+    And user "user1" has been added to group "Group2"
+    And user "user1" has been added to group "Group3"
+    And user "user1" has been added to group "Group4"
+    And user "user1" has been added to group "Group5"
+    And user "user1" has been added to group "Group6"
+    And user "user1" has been added to group "Group31"
+    And user "user1" has been added to group "A111111"
```
Comment (skshetry) [inline tests/acceptance/features/webUIAccount/accountInformation.feature]: I don't think you need this amount of group to demonstrate `multiple` groups.

File: tests/acceptance/stepDefinitions/accountContext.js (hunk 9 above)
Comment (jasson99) [reply tests/acceptance/stepDefinitions/accountContext.js]: done

**Pull request merged**